### DISCUSSION TITLE
How many nephews does Donald Duck have?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#594] [Galaxy Trivia] [answered]
**Bonus:** What are their names and the colors of their caps?

**Comment by Veercodeprog** [comment]
Huey, Dewey, and Louie are fictional characters created by the Walt Disney Company. Huey, Dewey, and Louie are usually depicted wearing identical outfits in different colors, with each wearing a cap that matches the color of their shirt. Huey wears a red shirt and cap, Dewey wears a blue shirt and cap, and Louie wears a green shirt and cap

Huey, Dewey, and Louie are known for their mischievous behavior and adventurous spirit. They are often depicted getting into trouble and causing chaos, but they also have a strong sense of loyalty and are always there to help their uncle Donald when he needs them.

**Comment by peterbaccaro** [comment, marked as answer]
Donald Duck has **three** nephews.  They features in an animated cartoon called Donald's Nephews that was released in1938.
His three nephews are called **Huey**, **Dewey**, and **Louie**.  They are distinguished by the colour of their caps; **Huey wears a red cap**, **Dewey wears a blue cap**, and **Louie wears a green cap**.

![ducks](https://user-images.githubusercontent.com/74666079/233288152-f26835a3-1dad-43cc-b690-e142c91bd485.png)
